### PR TITLE
KOGITO-1268 Businesskey support in process details

### PR DESCRIPTION
**Pull request by AjayJagan**
- added businessKey support to process details page

### CODE REVIEW

File: packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx [hunk 1]
```diff
@@ -90,6 +90,13 @@ const ProcessDetails: React.FC<IOwnProps> = ({ data, loading }) => {
               {data.ProcessInstances[0].processName}
             </Text>
           </FormGroup>
+          {data.ProcessInstances[0].businessKey && (
+            <FormGroup label="Businesskey" fieldId="businessKey">
```
Comment (cristianonicolai) [inline packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx]:
```suggestion
            <FormGroup label="Business Key" fieldId="businessKey">
```

**Review by cristianonicolai — COMMENTED** [review]
@AjayJagan there is an empty div between start and child process that is taking some space, can you check that, please.
Selection_046

File: packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx [hunk 2]
```diff
@@ -90,6 +90,13 @@ const ProcessDetails: React.FC<IOwnProps> = ({ data, loading }) => {
               {data.ProcessInstances[0].processName}
             </Text>
           </FormGroup>
+          {data.ProcessInstances[0].businessKey && (
+            <FormGroup label="Business key" fieldId="businessKey">
```
Comment (cristianonicolai) [inline packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx]:
```suggestion
            <FormGroup label="Business Key" fieldId="businessKey">
```

**Review by cristianonicolai — APPROVED** [review]

**Review by nmirasch — APPROVED** [review]

**Review by Sara4994 — APPROVED** [review]

**Review by srambach — APPROVED** [review]
👍

**Pull request merged**